### PR TITLE
Fix failing test compile for Spark 4.0.0

### PR DESCRIPTION
**Pull request by revans2**
This fixes https://github.com/NVIDIA/spark-rapids/issues/11350

### CODE REVIEW

**Comment by revans2** [comment]
build

**Review by razajafri — DISMISSED** [review]

File: tests/src/test/spark320/scala/com/nvidia/spark/rapids/ColumnCreationShim.scala [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+{"spark": "350"}
+{"spark": "351"}
+spark-rapids-shim-json-lines ***/
+package com.nvidia.spark.rapids
+
+import org.apache.spark.sql.Column
+import org.apache.spark.sql.catalyst.expressions.Literal
+
+object ColumnCreationShim {
+  def apply(lit: Literal): Column = new Column(lit)
```
[the remark this answers is not in the document]
Comment (revans2) [reply tests/src/test/spark320/scala/com/nvidia/spark/rapids/ColumnCreationShim.scala:47]: I thought I tried that,but I will try it again.

**Comment by revans2** [comment]
build

**Comment by revans2** [comment]
@razajafri and @jlowe please take another look

**Review by razajafri — APPROVED** [review]

**Review by jlowe — APPROVED** [review]

**Pull request merged**